content/Watsonx/WatsonxData/109.md

```diff
@@ -16,7 +16,7 @@ The TechZone watsonx.data environment comes pre-configured with presto engine, i
 
 3. In the **Add bucket** pop-up window, select/enter the following pieces of information
 
-> **Note:** If you have an IBM Cloud account, you can create your own bucket and use it in this lab. You can use the following bucket but make sure, you prefix your schema with your initials.
+> **Note:** If you have an IBM Cloud account, you can create your own bucket and use it in this lab. You can also use the following bucket but make sure, you prefix your schema name with your initials to avoid object conflict issue in bucket. 
 
   Bucket definition section
 
```

Comment:
Can we update the "Bucket name" on line 24 to reflect the bucket created for Chicago?  

In the future it probably makes sense to use a more general bucket name and recycle the credentials after each workshop.  Or just put **provided in workshop slack channel** if we would rather create a new bucket for each workshop.